build/gulp/ts.js

```diff
@@ -32,18 +32,22 @@ gulp.task('ts-vendor', function() {
         .pipe(gulp.dest(OUTPUT_ARTIFACTS_DIR));
 });
 
-gulp.task('ts-bundle', gulp.series(
+function bundleTS(jqueryPartFile) {
+    return gulp.src([jqueryPartFile].concat(TS_BUNDLE_SOURCES))
+        .pipe(concat('dx.all.d.ts'))
+        .pipe(headerPipes.bangLicense());
+}
 
+gulp.task('ts-bundle', gulp.series(
     function writeTsBundle() {
-        return gulp.src(TS_BUNDLE_SOURCES)
-            .pipe(concat('dx.all.d.ts'))
-            .pipe(headerPipes.bangLicense())
-            .pipe(gulp.dest(OUTPUT_ARTIFACTS_DIR)) // will be copied to the npm's /dist folder by another task
-            .pipe(replace('/*!', '/**'))
-            .pipe(replace(/\/\*\s*#StartGlobalDeclaration\s*\*\//g, 'declare global {'))
-            .pipe(replace(/\/\*\s*#EndGlobalDeclaration\s*\*\//g, '}'))
-            .pipe(replace(/\/\*\s*#StartJQueryAugmentation\s*\*\/[\s\S]*\/\*\s*#EndJQueryAugmentation\s*\*\//g, ''))
+        return bundleTS('./ts/dx.all-jquery.d.ts')
```

Comment:
I think it's better to use the word `augmentation`:
`dx.all.d.ts`
`dx.all.jquery-augmentations.d.ts`
```